mkosi.md

```diff
@@ -570,6 +583,22 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
   way is mounted into both the development and the final image while
   the package manager is running.
 
+`BaseTrees=`, `--base-tree=`
+
+: Takes a colon separated pair of directories to use as base images. When
```

Comment:
Only directories? Not images?

Reply:
I'd love for privileged and unprivileged mkosi to support the same set of features and disk image support here would break that. I figure you can always pack up your disk image as a tar as well and build your extension images using that as the base tree. Do you absolutely need to support building extensions against disk images?

Reply:
It's a bit of a pain to do, why not support both, and simply refuse if getuid != 0 if an image is passed?

Reply:
Added back disk image support.